### DISCUSSION TITLE
Updates question

### DISCUSSION THREAD

**Discussion by Bondarz** [LeDragoX/Win-Debloat-Tools#100] [Q&A] [answered]
Hello, I use the Win-Debloat-Tools-main for Windows 10 and Windows 11

1. is it necessary to restart the tools every few weeks/months? For example after major Windows updates? 
Or do I press debloat once?
2. if 1 is answered with yes: Should I download the .zip from github every few months, or does the script update itself after a start?
3. i used https://wpd.app/ before. Does it make sense to use both programs or are the debloat tools enough? 

I still need Word/Powerpoint, Teams and Windows updates.
As antivirus/firewall I use the Defender from https://github.com/AndyFul/ConfigureDefender

Thanks for your work!

**Comment by LeDragoX** [comment, marked as answer]
I'll answer you simple and clean.
1. You may want to use the debloat at least every major update (21H2 -> 22H1 for example). Nothing may change, but something could.
2. About the .zip, yes, if there are changes between the date you downloaded and the day you want to use it again. The script currently doesn't have an "auto-update" as its mainly the source code.
3. You can use whatever tool you want, as one tool may contain more tweaks than another, you'll only know by trying. I only tweak "the core stuff", to get a better experience through Windows.

Sorry if it removed Office 365 app, it came with Windows, and not everybody pay for it. About Teams, again, not everybody use it, majorly for meeting when you're on a job, but you can find it the "Software Install" tab.

Thanks for using the script btw :v
You being here shows how important this project is for me.
